src/plugins/experimentalWarnings/plugin.js

```diff
@@ -26,13 +27,8 @@ function showMessage(text, userSettingsKey, appHostFeature) {
         return Promise.resolve();
     }
 
-    return new Promise(function (resolve, reject) {
-        userSettings.set(userSettingsKey, '1', false);
-
-        import('alert').then(({default: alert}) => {
-            return alert(text).then(resolve, resolve);
-        });
-    });
+    userSettings.set(userSettingsKey, '1', false);
+    return alert(text);
```

Comment:
This no longer ignores exceptions like the original did.

Reply:
can you elaborate what needs to be done?

Reply:
i added a catch with a empty function, that's equivalent to the previous implementation.